packages/utils/odsp-doclib-utils/src/odspRequest.ts

```diff
@@ -3,8 +3,9 @@
  * Licensed under the MIT License.
  */
 
-import { fetch } from "./fetch";
-import { IOdspAuthRequestInfo, authRequestWithRetry } from "./odspAuth";
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
lint disables should have a comment explaining

Reply:
Don't think we need the disable.  (Removed)